Documentation/Rendering/ClippingPrimitive.md

```diff
@@ -0,0 +1,114 @@
+# Clipping Primitive
+
+The [`ClippingPrimitive`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingPrimitive) behaviors allow for performant [`plane`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingPlane), [`sphere`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingSphere), and [`box`](xref:Microsoft.MixedReality.Toolkit.Utilities.ClippingBox) shape clipping with the ability to specify which side of the primitive to clip against (inside or outside) when used with MRTK shaders.
+
+![primitive clipping gizmos](../../Documentation/Images/MRTKStandardShader/MRTK_PrimitiveClippingGizmos.gif)
+
+> [!Note]
```

Comment:
does this show up correctly in VS code markdown? Intuition tells me this has to be !NOTE vs !Note from past experience that I ran into before....maybe?